src/hacking/building-for-android.md

```diff
@@ -151,7 +151,7 @@ To get symbols resolved, you may need to provide additional library paths (at th
 
 OR you may need to enter the same path names as above in the support/android/apk/libs/armeabi/gdb.setup file.
 
-If you are not able to get past the "Application Servo (process com.mozilla.servo) is waiting for the debugger to attach." prompt after entering `continue` at `(gdb)` prompt, you might have to set Servo as the debug app (Use the "Select debug app" option under "Developer Options" in the Settings app).
+If you are not able to get past the "Application Servo (process com.servo.servoshell) is waiting for the debugger to attach." prompt after entering `continue` at `(gdb)` prompt, you might have to set Servo as the debug app (Use the "Select debug app" option under "Developer Options" in the Settings app).
```

Comment:
That's the current name ( changed in  https://github.com/servo/servo/pull/32554 ).

The organization name was changed from Mozilla to servo, to reflect the change in ownership / affiliation and the application we are shipping is called servoshell, so it does make sense to me. 
Now that I'm looking at it, perhaps we should also change the com domain ending to "org", since our domain is servo.org and not servo.com.

Reply:
ahh what i meant was, the other changes all say org.servo.servoshell but this one says com.servo.servoshell, was that intentional?

Reply:
aah, thanks for catching that! I was only looking at the highlighted portion of the diff, and completely missed that. Yes indeed it should be `org`.

Reply:
fixed!